website/docs/advanced/targeting/targeting.md

```diff
@@ -71,8 +71,16 @@ In this case, the feature flag evaluation is based on the secure hashes of the c
 
 | Comparator             | Description                                                                                                                    |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+|EQUALS| True if the _Comparison attribute_ equals the _Comparison value_.                                                                |
```

Comment:
Igazság szerint a contains, not contains-en kívül mindegyik hashed. Vagy mindegyiknél oda lehetne írni, vagy nem tudom. Csak fura, hogy csak az is one of is not one of-nál van ott.